administrator/components/com_j2store/views/apps/tmpl/default.php

```diff
@@ -186,7 +186,7 @@
 				<?php echo $this->pagination->getListFooter();?>
             </nav>
             <div class="text-center mt-2 mb-4 px-4">
-                <i class="fa-4x mb-2 fa-solid fa-cart-plus"></i>
+                <i class="fa-4x mb-2 fa-solid fas fa-cart-plus"></i>
```

Comment:
this should really be a span and not i
i can do a separate PR as this is across the entire repo if you want

Reply:
if you can clean up the whole repo for those items, that works for me. I am still getting used to the Joomla method so bear with me...